docs/app/Examples/modules/Dropdown/Usage/index.js

```diff
@@ -17,6 +17,12 @@ const DropdownUsageExamples = () => (
     />
 
     <ComponentExample
+      title='Close On Change'
+      description='A multiselect can close when the user changes its value.'
```

Comment:
Nit: for consistency with the props, can we go with `A multiple selection dropdown...` here?